tagController.js

```diff
@@ -0,0 +1,132 @@
+/*
+LANDWAR Unit Builder - V0.0.1
+author: Peter Roohr
+overview
+    + controller for TAG section of unit calculator
+*/
+
+
+/*
+    adds a new TAG row to the TAG table
+*/
+function tagAddRow(){
+    var tagTable = document.getElementById('tagTable');
```

Comment:
In modern JavaScript we use `let` to declare variables and const where possible.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Reply:
Though if a variable is used across multiple files use `var`

Reply:
not sure what you mean here.

Reply:
Yeah, rather than having numbers at the end of a variable name just stick them in an array. Roughly the same thing but then you can iterate over them. Less copypasta.